Package.swift

```diff
@@ -1,15 +1,11 @@
-// swift-tools-version:5.9
+// swift-tools-version:6.0
```

Comment:
self-review: this should be a separate PR. let's merge the fixes as a minor bump first and drop Swift 5.10 as a major bump.